src/main/java/core/basesyntax/transactionparser/CsvTransactionParser.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax.transactionparser;
+
+import core.basesyntax.FruitTransaction;
+import core.basesyntax.Operation;
+import java.util.ArrayList;
+import java.util.List;
+
+public class CsvTransactionParser implements TransactionParser {
+    private static final int OPERATION_INDEX = 0;
+    private static final int FRUIT_INDEX = 1;
+    private static final int QUANTITY_INDEX = 2;
+
+    @Override
+    public List<FruitTransaction> parseTransactions(List<String> lines) {
+        List<FruitTransaction> transactions = new ArrayList<>();
+        for (String line : lines) {
+            String[] transaction = line.split(",");
```

Comment:
All magic numbers, strings, and symbols in your code should be constants.
[this article](https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names)